python/lsst/sims/GalSimInterface/galSimInterpreter.py

```diff
@@ -96,6 +96,12 @@ class which tells the interpreter how to add sky noise to its images.
         self.centroid_list = []  # This is a list of the centroid objects which
                                  # will be written to the file.
 
+        # Make a trivial SED to use for faint things.
```

Comment:
This code block needs to be moved to `GalSimSiliconInterpeter.__init__` since this version of the class doesn't have a `.gs_bandpass_dict` attribute.

Reply:
Quite right.  Sorry.  Fixed.